api/controllers/WebActionsController.js

```diff
@@ -68,14 +68,22 @@ module.exports = {
    */
   join: function(req, res) {
     let redirectUrl;
-    
-    // if signing up through partner landing pages, 
+
+    // If signing up through partner landing pages,
     // redirect directly to confirmation page
+    // Else if user signs up through a campaign redirect to referral form.
 
     if (req.body.partner) {
-      redirectUrl = `/confirmation?phone=${req.body.phone}&firstName=${req.body.first_name}&partner=${req.body.partner}`;
+      redirectUrl = `/confirmation?phone=${req.body.phone}&firstName=${req.body
```

Comment:
😬  this is a pretty brutal line break from JSPrettier

Maybe there's another way we can format this to work? Possibly something like...

```
redirectUrl = `/confirmation`+
    `?phone=${req.body.phone}` +
    `&firstName=${req.body.first_name}` +
    `&partner=${req.body.partner}`;
```